src/slurm_plugin/clustermgtd.py

```diff
@@ -303,6 +307,7 @@ def __init__(self, config):
         self._compute_fleet_status_manager = None
         self._instance_manager = None
         self.set_config(config)
+        self.disable_nodes_on_insufficient_capacity = self._config.insufficient_capacity_timeout > 0
```

Comment:
Either you move this to the `set_config` function so that it gets updated when the config changes. Or we set this directly inside the config class.

nit (in case you leave it in this class): `_disable_nodes_on_insufficient_capacity`

Reply:
Move into ste_config function